client/public/locales/es/translation.json

```diff
@@ -36,15 +36,15 @@
   "Further Goals": "Objetivos pendientes",
   "No Goals": "Sin objetivos",
   "Loading goal…": "Cargando objetivo…",
-  "Click somewhere in the Lean file to enable the infoview.": "Pulsa en algún lugar del archivo Lean para habilitar la vista de información.",
-  "Waiting for Lean server to start…": "Esperando a que el servidor Lean se inicie…",
+  "Click somewhere in the Lean file to enable the infoview.": "Haz clic en el archivo para mostrar la vista de información.",
```

Comment:
"Pulsar" for click is simply wrong (or at least I've never heard anything like it).